sagemaker_neo_compilation_jobs/tensorflow_distributed_mnist/tensorflow_distributed_mnist_neo.ipynb

```diff
@@ -123,11 +119,12 @@
     "\n",
     "mnist_estimator = TensorFlow(entry_point='mnist.py',\n",
     "                             role=role,\n",
-    "                             framework_version='1.11.0',\n",
+    "                             framework_version='1.15.3',\n",
+    "                             py_version='py3',\n",
     "                             training_steps=1000, \n",
     "                             evaluation_steps=100,\n",
-    "                             train_instance_count=2,\n",
-    "                             train_instance_type='ml.c4.xlarge')\n",
+    "                             instance_count=2,\n",
```

Comment:
`TypeError: __init__() got an unexpected keyword argument 'instance_count'`

Reply:
I'm using SageMaker SDK 2.x. This version is backwards incompatible with 1.x. Are these notebooks supposed to work with 1.x as well?